.github/workflows/ci.yml

```diff
@@ -104,6 +104,11 @@ jobs:
             modin/config/__main__.py \
             modin/config/envvars.py \
             modin/config/pubsub.py
+      - run: |
+          python scripts/doc_checker.py \
```

Comment:
Why don't we check just modin/_compat?

Reply:
Because `doc_checker` tries to import each module, but some modules under `modin/_compat/core/py36` cannot be imported under our normal environment (certain imports are no longer in the same place in pandas 1.4 vs 1.1).

Un-importable code won't be executed on 3.8, so it's not a problem.
Completely fair solution would be to doc-check those separate files in 3.6 environment, though I'm not sure it's worth that.